modules/administration-guide/partials/proc_adding-or-removing-vscode-extensions-in-the-embedded-open-vsx-registry-instance.adoc

```diff
@@ -0,0 +1,83 @@
+:_content-type: PROCEDURE
+
+[id="adding-or-removing-vscode-extensions-in-the-embedded-open-vsx-registry-instance"]
+= Adding or removing pass:[<!-- vale RedHat.Spelling = NO -->] pass:[<!-- vale RedHat.Spelling = NO -->] VS Code pass:[<!-- vale RedHat.Spelling = YES -->] pass:[<!-- vale RedHat.Spelling = YES -->] extensions in the embedded Open VSX registry instance
+
+You can add or remove your choice of pass:[<!-- vale RedHat.Spelling = NO -->] pass:[<!-- vale RedHat.Spelling = NO -->] VS Code pass:[<!-- vale RedHat.Spelling = YES -->] pass:[<!-- vale RedHat.Spelling = YES -->] extensions in the embedded Open VSX registry instance that is cached by {prod-short} to support air-gapped, offline, and proxy-restricted environments.
+
+This will create a custom build of the Open VSX registry, which can be used in your organization’s workspaces. 
+
+NOTE: A subscription update reverts your `CheCluster` to default settings, and you must reapply your changes to `CheCluster`. 
+
+.Procedure
+
+include::example$snip_{project-context}-steps-to-add-or-remove-extensions-in-openvsx-sync-json.adoc[]
+
+** If the extension is published on link:https://open-vsx.org[open-vsx.org], you can add it by including an object with the extension `id` in the format __<published_by>__.__<unique_identifier>__ that you can collect from the extension page on link:https://open-vsx.org[open-vsx.org]:
```

Comment:
```suggestion
** If the extension is published on link:https://open-vsx.org[open-vsx.org], you can include it using the extension `id` in the format __<published_by>__.__<unique_identifier>__. You can find the correct `id` from the extension's listing page on link:https://open-vsx.org[open-vsx.org]:
```

Reply:
@nickboldt, I'd like to tweak your suggestion. Is this okay? 
```suggestion
** If the extension is published on link:https://open-vsx.org[open-vsx.org], you can add it by including its extension `id` in the format __<published_by>__.__<unique_identifier>__. You can find these details on the extension's listing page on link:https://open-vsx.org[open-vsx.org].
```

Reply:
since we only care about the `id`, why make the statement more vague with `these details` when the only detail you want is the `id` ?